ResourceStatusSystem/Agent/EmailAgent.py

```diff
@@ -25,8 +25,11 @@ def __init__( self, *args, **kwargs ):
     AgentModule.__init__( self, *args, **kwargs )
     self.diracAdmin = None
     self.default_value = None
-    self.cacheFile = None
-    self.cacheFile = os.getenv('DIRAC') + 'work/ResourceStatus/cache.json'
+
+    if os.getenv('DIRAC'):
```

Comment:
why is it prefered? If there is an environmental variable named `DIRACwork` (maybe in the future) then the `if DIRAC in os.environ` will return `True` and the `os.getenv('DIRAC')` will fail, right?